Configuration/DuckDuckGoDeveloper.xcconfig

```diff
@@ -21,3 +21,9 @@ APP_ID = com.duckduckgo.mobile.ios
 
 // A prefix for group ids. Must start with "group.".
 GROUP_ID_PREFIX = group.com.duckduckgo
+
+// The code signing style needs to be specified manually because the DDG team has multiple provisioning profiles that match the team and bundle identifier
+CODE_SIGN_STYLE = Manual
+
+// The manualy specified provisioning profile
+PROVISIONING_PROFILE_SPECIFIER = "iOS 14 - Dev - App"
```

Comment:
One quick request here: this'll include the quotes themselves in the provisioning profile specifier setting, after taking these out then this should work great!

```suggestion
PROVISIONING_PROFILE_SPECIFIER = iOS 14 - Dev - App
```